watertap/flowsheets/generic_desalination_train/generic_train.py

```diff
@@ -20,42 +20,37 @@
     NonNegativeReals,
     units as pyunits,
 )
+from pyomo.network import Arc
+from pyomo.util.check_units import assert_units_consistent
+
 from idaes.core import (
     FlowsheetBlock,
 )
-
 import idaes.core.util.scaling as iscale
 from idaes.core.util.model_statistics import degrees_of_freedom
-
-
-from watertap.core.solvers import get_solver
-
-from watertap.property_models.multicomp_aq_sol_prop_pack import (
-    MCASParameterBlock,
-    ActivityCoefficientModel,
-    DensityCalculation,
-    MaterialFlowBasis,
-)
-
 from idaes.core.util.initialization import propagate_state
-
 from idaes.models.unit_models import (
     Product,
     Feed,
 )
 
 import watertap.flowsheets.generic_desalination_train.utils.scale_utils as scaleTools
-from pyomo.util.check_units import assert_units_consistent
 from watertap.flowsheets.generic_desalination_train.costing import (
     generic_costing,
     stream_costing,
 )
-from pyomo.network import Arc
 from watertap.flowsheets.generic_desalination_train.unit_operations import (
     desalter,
     mixer,
     separator,
 )
+from watertap.property_models import (
+    MCASParameterBlock,
+    ActivityCoefficientModel,
+    DensityCalculation,
+    MaterialFlowBasis,
```

Comment:
No big deal, but I noticed that in another file change, you switched to importing `MaterialFlowBasis` from `idaes.core`.  That said, I prefer having `MaterialFlowBasis` importable from `watertap.property_models ` too for convenience.

Reply:
I agree and I was going to do this but thought I would get pushback because it would require doing `from idaes.core import MaterialFlowBasis` in the init and I am not sure if that is a packaging no-no. But I can add it